File: skopintseva_may/24.py
with open("24.txt") as f:
    count = 0
    mx = 0
    s = f.readline()
    i = 0
    while(i < len(s) - 1):
        if s[i] == s[i + 1]:
            count += 2
            i += 1
        else:
            if count > mx:
                mx = count
            count = 0
        i += 1
    print(mx)
        
        

File: skopintseva_may/24.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
	std::string s;
	int count = 0, mx = 0;
	std::ifstream f("24.txt");
	getline(f, s);
	for(int i = 0; i < s.size() - 1; i++) {
		if(s[i] == s[i + 1]) {
			count += 2;
			i += 1;
		} else {
			if(count > mx)
				mx = count;
			count = 0;
		}
	}
	std::cout << mx;
	return 0;
}
